addons/io/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/accessories/HomekitGarageDoor.java

```diff
@@ -0,0 +1,170 @@
+package org.openhab.io.homekit.internal.accessories;
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.